soapclient/SforceEnterpriseClient.php

```diff
@@ -50,7 +50,8 @@ function SforceEnterpriseClient() {
    * @return SaveResult
    */
   public function create($sObjects, $type) {
-    foreach ($sObjects as &$sObject) {
+    $arg = []

```

Comment:
I accidentally deleted the branch of this pull request so I couldn't correct this typo.
I have created a new pull request #49